brainstormforce/ultimate-addons-for-gutenberg
# Fix: Post Layout - Removed space from selector.

### Description

- Removed added space for tablet and mobile.
- Updated selector class for mobile media query. 

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and Drop Post Layout
- Update or change Columns settings as per the responsive device. (columns value changes as per post type.)
- Check Frontend as well as Backend.

### Checklist:
